images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -125,12 +125,6 @@ function Get-SQLPSVersion {
     return "SQLPS $version"
 }
 
-function Get-SQLServerPSVersion {
-    $module = Get-Module -Name SQLServer -ListAvailable
-    $version = $module.Version
-    return "SQLServer PS $version"
-}
-
```

Comment:
Why do you remove this method? This version is now missing in the docs

Reply:
It's already included to "Powershell Modules" section

```
#### Powershell Modules
| Module             | Version                     |
| ------------------ | --------------------------- |
| DockerMsftProvider | 1.0.0.8                     |
| MarkdownPS         | 1.9                         |
| Pester             | 3.4.0<br>5.0.4              |
| PowerShellGet      | 1.0.0.1<br>2.2.4.1<br>2.2.5 |
| PSWindowsUpdate    | 2.2.0.2                     |
| SqlServer          | 21.1.18228                  |
| VSSetup            | 2.2.16                      |
```

Reply:
thanks!